sys/prebuilt-bindings/geos_3.10.rs

```diff
@@ -0,0 +1,4555 @@
+/* automatically generated by rust-bindgen 0.60.1 */
+
+pub const GEOS_VERSION_MAJOR: u32 = 3;
```

Comment:
This is what I told you about: bindgen doesn't generate libc types, which means that they can become completely invalid if you switch to another arch.